### DISCUSSION TITLE
How I can set up this with Packer?

### DISCUSSION THREAD

**Discussion by MannyFay** [adalessa/laravel.nvim#47] [Q&A]
Has it anybody tried with Packer? I don't get it to run :/

**Comment by adalessa** [comment]
Sorry for the delay @MannyFay , can you provide more details, like the configuration that you tried?.
I would assume that you only need to install it with packer and later call the setup method for it, and you will have to bind the keybindings, since with the plugins does not set them up, you should have access to the commands.
